tests/src/test/scala/whisk/core/admin/WskAdminTests.scala

```diff
@@ -33,7 +34,26 @@ import whisk.core.entity.Subject
 import common.TestUtils
 
 @RunWith(classOf[JUnitRunner])
-class WskAdminTests extends TestHelpers with Matchers {
+class WskAdminTests extends TestHelpers with Matchers with BeforeAndAfterAll {
+
+  override def beforeAll() = {
+    val wskadmin = new RunWskAdminCmd {}
```

Comment:
you could just delete all the test ids and tolerate the failure (vs checking if they exist first).
i used this particular approach in another test suite recently to clean up ids.

Reply:
Sure, thanks for reviewed! So enclose within a `Try` like this?